apps/web-giddh/src/app/accounting/invoice-grid/invoice-grid.component.html

```diff
@@ -20,18 +19,45 @@
             </div>
 
             <!-- invoice num -->
-            <div class="clearfix form-inline mrB">
+            <div class="clearfix form-inline mr-b05">
                 <div class="mx-350 form-group" [hidden]="isSalesInvoiceSelected">
                     <!-- decimalDigitsDirective -->
                     <label>{{ invoiceNoHeading }}:</label> <input type="text" placeholder="Invoice no."
-                        [(ngModel)]="data.invoiceNumberAgainstVoucher" class="form-control max-height28 appearance_none"
-                        style="max-width:200px" [disabled]="isSalesInvoiceSelected" />
+                        [(ngModel)]="data.invoiceNumberAgainstVoucher" class="form-control max-height28 appearance-none invoice-no-width" [disabled]="isSalesInvoiceSelected" />
                 </div>
 
                 <!-- key not available in request
     <div class="mx-300 form-group" (clickOutside)="showInvoiceDate=false;">
     <span>Date: </span>
-    <input type="text" [ngModel]="moment(entryDate).format(giddhDateFormat)" decimalDigitsDirective readonly (focus)="showInvoiceDate = true;" class="form-control appearance_none max-height28" />
+<<<<<<< HEAD
```

Comment:
Conflicted code is committed please check throughout the PR once

Reply:
done